content/docs/Getting started/deploying-cuebot.md

```diff
@@ -84,6 +84,20 @@ Make sure you also complete the following steps:
     to the database. This configuration is outside of the scope of this
     guide.{{% /alert %}}
 
+1.  Cuebot needs to know the location of the filesystem where render assets
+    are stored and log files are written. In a large-scale deployment a shared
+    filesystem such as NFS is often used for this purpose. For the purposes of
+    this tutorial, you can use any directory you have write access to.
+
+    ```shell
+    export CUE_FS_ROOT="$HOME/opencue-tutorial"
```

Comment:
Here and in the RQD installation guide, I find the use of 'tutorial' a little unusual. As an admin, I would expect to be following this install guide to set up my system for testing or production. Consider `/opencue-testing` or `/opencue-demo` ?

Part of my rationale is that it might one day be good to have a proper end-to-end tutorial :)

Reply:
Makes sense, changed to `opencue-demo`.